### PR TITLE
Add new article: Seamless Playback of Media Segments in Append Order

### PR DESCRIPTION
**Pull request by samdutton**
As per issue #2674.

### CODE REVIEW

File: src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+published_on: 2016-03-10
+updated_on: 2016-03-10
+authors:
+  - samdutton
+tags:
+  - audio
+  - chrome50
+  - media
+  - mse
+  - news
```
Comment (PaulKinlan) [inline src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown]: remove

File: src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+<p class="intro">The HTML audio and video elements enable you to load, decode and play media, simply by providing a src URL:
+</p>
+
+{% highlight html %}
+<video src='foo.webm'></video>
+{% endhighlight %}
+
+That works well in simple use cases, but for techniques such as [adaptive streaming](https://www.youtube.com/watch?v=Fm3Bagcf9Oo), the Media Source Extensions API (MSE) provides more control. MSE enables streams to be built in JavaScript from segments of audio or video.
+
+You can try out MSE at [simpl.info/mse](https://simpl.info/mse):
```
Comment (PaulKinlan) [inline src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown]: Any chance we can put htis in cHrome Samples?

File: src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+<a href="https://simpl.info/mse"><img class="screenshot" src="/web/updates/images/2016/03/mse-sourcebuffer/screenshot.jpg" alt="Screenshot of video played back using the MSE API"></a>
```
Comment (PaulKinlan) [inline src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown]: This is MASSSIVE, please scale back or remove image. It looks out of place.

File: src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+For all stream types, you can change the value from _segments_ to _sequence_. This means segments will be played back in the order they were appended, and new timestamps generated accordingly:
+
+{% highlight javascript %}
+sourceBuffer.mode = 'sequence';
+{% endhighlight %}
+
+Being able to set the `mode` value to _sequence_ ensures continuous media playback, no matter if the media segment timestamps were discontinuous — for example, if there were problems with video [muxing](https://en.wikipedia.org/wiki/Multiplexing), or if (for whatever reason) discontinuous segments are appended. It is possible for an app to polyfill with [`timestampOffset`](https://developer.mozilla.org/en-US/docs/Web/API/SourceBuffer/timestampOffset)) to ensure continuous playback, if correct stream metadata is available, but _sequence_ mode makes the process simpler and less error prone.
```
Comment (PaulKinlan) [inline src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown]: There is an errant ) here.

**Comment by PaulKinlan** [comment]
Some of this needs to be addressed, you can see a preview here: https://pr-2691-dot-weasel-dot-web-central.appspot.com//web/updates/2016/03/mse-sourcebuffer?hl=en

File: src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown (hunk 1 above)
Comment (samdutton) [inline src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown]: remove 'news'?

File: src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown (hunk 2 above)
Comment (samdutton) [inline src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown]: I'd like to do that, but I don't think I'll have time today. Agree it's not ideal linking to external samples, but at least it has a nice short URL...

File: src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown (hunk 3 above)
Comment (samdutton) [inline src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown]: Agree! Scaled.

File: src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown (hunk 4 above)
Comment (samdutton) [inline src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown]: Good spot — fixed.

**Pull request merged**